src/dodal/devices/pressure_jump_cell.py

```diff
@@ -0,0 +1,236 @@
+from enum import Enum
+
+from ophyd_async.core import (
+    ConfigSignal,
+    DeviceVector,
+    SignalR,
+    StandardReadable,
+)
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+
+class PumpState(str, Enum):
+    MANUAL = "Manual"
+    AUTO_PRESSURE = "Auto Pressure"
+    AUTO_POSITION = "Auto Position"
+
+
+class BusyState(str, Enum):
+    IDLE = "Idle"
+    BUSY = "Busy"
+
+
+class TimerState(str, Enum):
+    TIMEOUT = "TIMEOUT"
+    COUNTDOWN = "COUNTDOWN"
+
+
+class StopState(str, Enum):
+    CONTINUE = "CONTINUE"
+    STOP = "STOP"
+
+
+class ValveControlRequest(str, Enum):
+    OPEN = "Open"
+    CLOSE = "Close"
+    RESET = "Reset"
+    ARM = "Arm"
+    DISARM = "Disarm"
+
+
+class PumpMotorControlRequest(str, Enum):
+    ENABLE = "Enable"
+    DISABLE = "Disable"
+    RESET = "Reset"
+    FORWARD = "Forward"
+    REVERSE = "Reverse"
+
+
+class PumpMotorDirectionState(str, Enum):
+    ZERO = "0"
+    FORWARD = "Forward"
+    REVERSE = "Reverse"
+    THREE = "3"
+    FOUR = "4"
+    FIVE = "5"
+    SIX = "6"
+    SEVEN = "7"
+
+
+class ValveState(str, Enum):
+    FAULT = "Fault"
+    OPEN = "Open"
+    OPENING = "Opening"
+    CLOSED = "Closed"
+    CLOSING = "Closing"
+
+
+class FastValveState(str, Enum):
+    FAULT = "Fault"
+    OPEN = "Open"
+    OPEN_ARMED = "Open Armed"
+    CLOSED = "Closed"
+    CLOSED_ARMED = "Closed Armed"
+    NONE5 = "Unused"
+
+
+class LimitSwitchState(str, Enum):
+    OFF = "Off"
+    ON = "On"
```

Comment:
I think all of the enums should be prepended with the relevant device class, so if you import multiple e.g. `PumpState` it's clearer which is for the `PressureJumpCell` and which is for the `PeristalticPump` if one is `PressureJumpPumpState` and the other is `PeristalticPumpState`

Reply:
when it comes to it at i22 we can sort it out then

Reply:
resolved elsewhere